internal/devicescale/impl_desktop_unix.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build (dragonfly || freebsd || linux || netbsd || openbsd || solaris) && !android && !ios && !js
+// +build dragonfly freebsd linux netbsd openbsd solaris
+// +build !android
+// +build !ios
+// +build !js
```

Comment:
Actually the original code listed Unix (or Unix-like) systems, but this is fragile. What about

```
//go:build !android && !ios && !js && !plan9 && !windows
```

?

Reply:
Done.